src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -35,6 +35,7 @@ interface PharmacyProps {
   website: string;
   lastUpdated: string;
   vaccineAvailabilities: VaccineAvailabilitiesByDateAndRequirementsInterface;
+  organizationId: number;
```

Comment:
Unless I'm mistaken, organizations are optional - if so, this should be an optional prop: `organizationId?: number`